src/ir/IRInlineHandler.ts

```diff
@@ -1,82 +1,102 @@
 import { DisallowedArgError, ErrorLogger } from "..";
 import { InlineHandler, Sugar, SugarsByStart } from "../parser/InlineHandler";
 import { InputPosition } from "../parser/InputPosition";
-import { HMError } from "../schema/errors";
+import {
+	DisallowedInArgError,
+	DisallowedInHeadError,
+	UnknownInlineTagError
+} from "../schema/errors";
 import { InlineSchema, Schema } from "../schema/Schema";
+import { InlinePropDefinition } from "../schema/SchemaDefinition";
 import { last } from "../utils";
 import { emptyBlockProps, IRNode, IRNodeList } from "./IRNode";
 
 export class IRInlineHandler implements InlineHandler {
-	protected readonly inlineGroupStack: IRNodeList[] = [];
-	protected readonly inlineElementStack: Array<{ node: IRNode; schema: InlineSchema }> = [];
-	protected ignoreFlag: boolean = false;
+	protected readonly inlineGroupStack: Array<{
+		nodeList: IRNodeList;
+		schema: InlinePropDefinition;
```

Comment:
With my proposition to add an `ArgSchema` class, we could replace this with `ArgSchema` and have `parent.schema.tag` be available :)

Reply:
Done ✅